desc/objectives/objective_funs.py

```diff
@@ -2,12 +2,13 @@
 from abc import ABC, abstractmethod
 from inspect import getfullargspec
 
-
+import warnings
+from termcolor import colored
 from desc.backend import use_jax, jnp, jit
 from desc.utils import Timer
 from desc.io import IOAble
 from desc.derivatives import Derivative
-from desc.compute import arg_order
+from desc.compute import arg_order, compute_jacobian
```

Comment:
don't need this import anymore. Also not sure about warnings and colored

Reply:
also unneeded